src/cdm/open_cdm_platform_common.h

```diff
@@ -32,7 +40,11 @@ enum PLATFORM_CALL_STATE {
 
 struct PlatformResponse {
   PLATFORM_CALL_STATE platform_response;
+#ifdef WPE
+  uint32_t sys_err;
+#else	// Chrome
   int32 sys_err;
```

Comment:
We don't want a difference here on the declaration of sys_err variable between WPE and Chromium. 
This change to 'uint32_t sys_err in WPE fork was introduced by commit

Author: wouterlucas <wouter@wouterlucas.com>  2017-02-01 21:41:46
Committer: wouterlucas <wouter@wouterlucas.com>  2017-02-01 21:41:46
Parent: 4b415d52b930e62d6339de7de9af899353c8fb30 (RPC interfaces updated and extended)
Child:  458a9dedcb945401b4e564703b54bf6268a1d3a9 (SetServerCertificate Implementation added)
Branches: remotes/origin/cturner/maintenance, remotes/origin/cturner/playready-leak, remotes/origin/wpe, remotes/origin/wpe-master, wpe
Follows: 1.0.0.0
Precedes: opencdm-20170506-before-removing-v1-v2-playready

    Initial checking of changes from Haseenamol

 This variable is only ever assigned from platform_val 

src/com/common/rpc/opencdm_xdr.h
struct rpc_response_generic {
        int platform_val;
}

which is an int, so int32 seems more correct than uint32_t.